src/test/java/MostCommonTest.java

```diff
@@ -0,0 +1,27 @@
+import org.example.MostCommon;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MostCommonTest {
+    MostCommon words;
+
+    @BeforeEach
+    void setUp() {
+        words = new MostCommon();
+    }
+
+    @Test
+    @DisplayName("Read from a given text file should work")
+    void displayMostCommon() {
+        assertNotNull(words.getResult("movie.txt"));

```

Comment:
It's good to assert how many words are you exactly expecting. Otherwise you don't test, whether the algorithm is actually working.